### DISCUSSION TITLE
SpookVooper API - v2.0.0 Release Masterthread

### DISCUSSION THREAD

**Discussion by identithree** [bowlingballindustries/spookvooper-api#12] [Updates]
#  SpookVooper API - v2.0.0 Release Masterthread
This is the masterthread for all discussions about SVAPI v2-alpha2
**Credits:** @bluebeargreen-2 @brndnln 

## Q&A
Q: When will it release?
A: Soon™ (We need to add more features at the current time)  
  
Q: What features will we have access to?
A: You will have access to all the features from before the update.  
  
Q: Will the API have more features than [SpikeViper/SpookVooper.Api](https://github.com/SpikeViper/SpookVooper.Api)?
A: Yes, this API will have support for the ExchangeHub and NVSE later down the line, once I figure out how Jacob functions. That might have to happen in 2.1.0  
  
Q: How do I get this?
A: It will be avalible on GitHub's Package Registry. You should be able to type `npm i @vexico/spookvooper-api` (theoretically). If you used to use NPM's spookvooper-api, please uninstall it by using `npm r spookvooper-api`.  
  
Q: If I get this through the Releases tab, will you support installing it?
A: No. This would take a while to support all the ways to get it. The only (supported) way to get it would be to use it through a package manager.

## How do I use this?
Start by downloading the release once it goes live in the [Releases tab](https://github.com/vexico/spookvooper-api/releases) or get it through the GitHub Package Registry

**If you are using Typescript:**  
You can just import it. Typings are included in the module.
```ts
import * as SVAPI from '@vexico/spookvooper-api'
// OR
import { User, Group, Disctrict /* etc... */ } from '@vexico/spookvooper-api'
```  
  
**If you are using Javascript:**
You can use require/exports or import/export.
```js
/*
  CommonJS (require/exports)
*/

const SVAPI = require('@vexico/spookvooper-api')
// OR
const { User, Group, District /* etc... */ } = require('@vexico/spookvooper-api')


/*
  ES6 Modules (import/export)
*/

import * as SVAPI from '@vexico/spookvooper-api'
// OR
import { User, Group, Disctrict /* etc... */ } from '@vexico/spookvooper-api'
```

**If you are Jacob:**  
Please, oh god please, don't be dumb. 

## Developer Outreach
What are you planning on doing with this? What else would you like added to this API wrapper?  
Reply below! 

## Closing
Hope you enjoy the things that we make, just wait to see what we've got planned with [VexiBot](https://github.com/vexico/VexiBot)  
  
Happy Hacking!
 \- Brendan Lane

**Comment by fruitcake5** [comment]
pp man

**Reply by identithree** [reply]
mmm

**Comment by superjacobl** [comment]
Python support when?

**Reply by identithree** [reply]
feck off 🙃

**Reply by fruitcake5** [reply]
> feck off 🙃

I second this
